### ISSUE TITLE
Move to android.arch.lifecycle:viewmodel

### ISSUE THREAD

**Issue by dkrivoruchko** [Ekito/koin#42]
Starting from [1.1.0](https://developer.android.com/topic/libraries/architecture/release-notes.html) new, much smaller dependencies are available: android.arch.lifecycle:viewmodel:1.1.0.
Please use it instead of android.arch.lifecycle:extensions:1.1.0

**Comment by arnaudgiuliani** [comment]
Hello,

Koin needs `ViewModelStores` to make the bridge between Android Arch ViewModel lifecycle & Koin instances factory. As I see, It is in `android.arch.lifecycle:extensions`

**Comment by dkrivoruchko** [comment]
Ok, Thanks for great lib!

**Comment by arnaudgiuliani** [comment]
Thanks!

**Comment by dkrivoruchko** [comment]
There is an update with more granular dependencies.
[https://developer.android.com/topic/libraries/architecture/adding-components.html](https://developer.android.com/topic/libraries/architecture/adding-components.html)

Is it now possible to use smaller dependencies?

**Comment by arnaudgiuliani** [comment]
Let me check that.

**Comment by dkrivoruchko** [comment]
It's a pity

**Comment by arnaudgiuliani** [comment]
yes, ViewModel extension APi are effectively in `extensions` sorry

**Comment by arnaudgiuliani** [comment]
But ... I give a try on limiting imports with gradle

**Comment by Apisov** [comment]
Yes, I think excluding `livedata` from `extensions` import would work

Something like this:
```Groovy
implementation("android.arch.lifecycle:extensions:${Versions.archLifecycle}") {
       exclude module: 'livedata'
}
```

**Comment by arnaudgiuliani** [comment]
Hello, fixed in `0.9.2-alpha-6` 👍
